js/main.js

```diff
@@ -0,0 +1,17 @@
+const nav = document.querySelector('.nav-mobile');
+const navBtn = document.querySelector('.burger-btn');
```

Comment:
This doesn't catch only navButton. If there's any other object with this class, then it would be used as well. I think it would be better to use a bit more specific locator.

Reply:
I will use this burger-btn only in nav. I created it only for mobile nav.